bib/cmd/bootc-image-builder/image.go

```diff
@@ -78,7 +78,8 @@ func manifestForDiskImage(c *ManifestConfig, rng *rand.Rand) (*manifest.Manifest
 
 	img := image.NewBootcDiskImage(containerSource)
 	img.Users = users.UsersFromBP(customizations.GetUsers())
-	img.Groups = users.GroupsFromBP(customizations.GetGroups())
```

Comment:
I will add this back once https://github.com/osbuild/images/pull/593 is merged

Reply:
The PR is in the queue and will probably be merged in a few mins but we're also going to need a release and dependency update so I think it's okay to merge this and open a follow-up immediately after.

Reply:
Groups are back now too, I close this conversation.